src/commonMain/kotlin/io/github/detekt/sarif4k/Merging.kt

```diff
@@ -0,0 +1,48 @@
+@file:JvmName("Merging")
+package io.github.detekt.sarif4k
```

Comment:
> @BraisGabin It seems that multiplatform doesn't like @JvmName.

According to https://slack-chats.kotlinlang.org/t/13888455/does-file-jvmname-work-in-commonmain-or-is-it-expected-to-wo#e6740d8b-28d3-4029-bb82-ea5ee82e5cfb

```suggestion
@file:JvmName("Merging")
package io.github.detekt.sarif4k

import kotlin.jvm.JvmName
```

but I guess, maybe it isn't such a big deal and just becomes inconsistent with @JsName and @NativeName?